package.json

```diff
@@ -12,25 +12,38 @@
     "packages/*"
   ],
   "scripts": {
-    "start": "lerna run dev --parallel",
-    "dev": "lerna run dev --parallel",
-    "lint": "lerna run lint --stream",
-    "test": "lerna run test --stream",
-    "test:qa": "lerna run test:qa --stream",
-    "version:safe": "lerna version --no-git-tag-version --no-push",
-    "pre-commit": "lerna run --concurrency 1 --stream precommit --since HEAD --exclude-dependents",
-    "toc": "markdown-toc README.md -i",
-    "prepare": "husky install",
-    "postinstall": "node node_modules/puppeteer/install.js"
+    "build": "npm run build --workspace=autocannon-ui-frontend",
+    "lint": "npm run lint --stream --workspaces",
+    "pre-commit": "npm run --concurrency 1 --stream precommit --since HEAD --exclude-dependents --workspaces",
```

Comment:
this should no longer be needed because husky scripts are executed via other means, not via npm scripts anymore